src/service/MailchimpSyncIntegrationService/MailchimpService.Types.ts

```diff
@@ -4,7 +4,10 @@ export namespace MailchimpServiceTypes {
     }
 
     export interface MailchimpLists {
-        lists: MailchimpList[];
+        entries: MailchimpList[];
+        page: number;
+        size: number;
+        totalCount: number;
     }
```

Comment:
And maybe rename `MailchimpList` to `MailchimpItem`. The suffix "List" seems completely wrong :D 

```suggestion
    import {PagedResponse} from '@flowfact/types';
     export type MailchimpLists = PagedResponse<MailchimpList>;
```

Reply:
Actually MailchimpList is correct value because it represents Mailchimp List Item
https://mailchimp.com/developer/marketing/api/lists/

Reply:
Then name it MailchimpListItem :)